src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import messages from '../messages';
+import PopupBase from '../PopupBase';
+import { Options, PopupReference } from '../Popper';
+
+import './PopupList.scss';
+
+export type Props = {
+    reference: PopupReference;
+};
+
+const PopupList = ({ reference, ...rest }: Props): JSX.Element => {
+    const options: Partial<Options> = {
```

Comment:
Can this be moved to a constant outside the component? The component itself can use implicit return.